pkg/apis/kops/v1alpha2/componentconfig.go

```diff
@@ -116,6 +116,9 @@ type KubeletConfigSpec struct {
 	// image garbage collection is never run. Lowest disk usage to garbage
 	// collect to.
 	ImageGCLowThresholdPercent *int32 `json:"imageGCLowThresholdPercent,omitempty" flag:"image-gc-low-threshold"`
+	// imagePullProgressDeadline is the timeout for image pulls
+	// If no pulling progress is made before this deadline, the image pulling will be cancelled. (default 1m0s)
+	ImagePullProgressDeadline *metav1.Duration `json:"imagePullProgressDeadline,omitempty" flag:"image-pull-progress-deadline"`
```

Comment:
I believe the flag was introduced in >= 1.6 .. perhaps check in validation.go, or perhaps it 1.5 is old enough not to count cc @chrislovecnm ..

Reply:
hi @izakp .. i meant that if someone tried to set it but was using a <= 1.6 version it would fail with a not-supported message. Though a quick glance through the code I can't see us doing else where and 1.5 is probably old enough not to worry too much .. So we can let this one slide ..

Reply:
We don't typically validate the flags if you set them directly (I think, anwyay!).  It's obviously great if you want to add validation, but there are _many_ ways to break your cluster once you start setting flags directly :-)